scripts/deploy-ebook-to-www

```diff
@@ -310,8 +310,11 @@ do
 			# We do this first because the tweaks below shouldn't apply to the single-page file
 			se recompose-epub --xhtml --output "${workDir}"/single-page.xhtml --extra-css-file="${webRoot}/css/web.css" "${workDir}"
 
+			# Wrap book contents in a <main> tag (and extra <div> for styling reasons)
+			sed --in-place --regexp-extended "s|<body([^>]*)>|<body><main\1><div>|; s|</body>|</div></main></body>|" "${workDir}"/single-page.xhtml
+
 			# Add a navbar with a link back to the homepage
-			sed --in-place --regexp-extended "s|<body(.*?)>|<body\1><header><nav><ul><li><a href=\"/\">Standard Ebooks</a></li><li><a href=\"${bookUrl}\">Back to ebook</a></li></ul></nav></header>|" "${workDir}"/single-page.xhtml
```

Comment:
sed doesn't actually support non-greedy regex matching (https://stackoverflow.com/questions/1103149/non-greedy-reluctant-regex-matching-in-sed) and so this would incorrectly insert the header after all the tags that follow body on the same line. Previously, body was the only opening tag on its line so this wouldn't matter.